src/libpmemfile/preload.c

```diff
@@ -60,6 +60,29 @@
 #include "libpmemfile-core.h"
 #include "util.h"
 
+/*
+ * Some syscalls that are missing on older kernels.
+ */
+#ifndef SYS_renameat2
+#define SYS_renameat2 316
+#endif
+
+#ifndef SYS_execveat
+#define execveat 322
+#endif
+
+#ifndef SYS_copy_file_range
+#define SYS_copy_file_range 326
+#endif
+
+#ifndef SYS_preadv2
+#define SYS_preadv2 327
+#endif
+
+#ifndef SYS_pwritev2
+#define SYS_pwritev2 328
+#endif
```

Comment:
This list is architecture-specific. Please surround it with `#if defined(__x86_64__)`. If someone will try to compile it on different architecture he will get nice compile error.